03-pipefilter.md

```diff
@@ -107,11 +107,12 @@ $ wc -l *.pdb > lengths.txt
 ~~~
 
 The greater than symbol, `>`, tells the shell to **redirect** the command's output
-to a file instead of printing it to the screen.
-The shell will create the file if it doesn't exist,
-or overwrite the contents of that file if it does.
-(This is why there is no screen output:
-everything that `wc` would have printed has gone into the file `lengths.txt` instead.)
+to a file instead of printing it to the screen. The shell will create 
+the file if it doesn't exist. In case the file exists, it will be 
+silently overwritten, which may lead to data loss and thus requires
+some caution. (This is why there is no screen output:
```

Comment:
I might move this parenthetical note to after "...printing it to the screen." in line 110.